IPython/terminal/magics.py

```diff
@@ -26,7 +26,7 @@ def get_pasted_lines(sentinel, l_input=py3compat.input, quiet=False):
         prompt = ""
     while True:
         try:
-            l = py3compat.str_to_unicode(l_input(prompt))
```

Comment:
str_to_unicode is a no-op on Python 3, and should be replaced with no call at all, rather than casting `str` to `str`.